models/business-unit/src/company/types.ts

```diff
@@ -16,7 +16,6 @@ export type TCompanyDraftGraphql = TCompanyDraft;
 export type TCompany = Company;
 export type TCompanyGraphql = Omit<TCompany, 'topLevelUnit' | 'parentUnit'> & {
   ancestors: [];
-  inheritedStores: null;
```

Comment:
I removed this type from here since it's already defined in Company type from the platform SDK